src/types/tokens.rs

```diff
@@ -88,11 +151,25 @@ impl std::fmt::Display for FTBalance {
     }
 }
 
+/// Account balance on the NEAR blockchain.
+///
+/// This balance doesn't include staked NEAR tokens or storage
 #[derive(Debug, Clone, PartialEq, Default, Eq, Serialize, Deserialize)]
 pub struct UserBalance {
-    pub liquid: NearToken,
-    pub locked: NearToken,
+    /// The total amount of NEAR tokens in the account.
+    ///
+    /// Please note that this is the total amount of NEAR tokens in the account, not the amount available for use.
+    pub total: NearToken,
+    /// The amount of NEAR tokens locked in the account for storage usage.
+    pub storage_locked: NearToken,
+    /// The storage usage by the account in bytes.
     pub storage_usage: u64,
+    /// The amount of NEAR tokens staked on a protocol level.
+    /// Applicable for staking pools only in 99.99% of the cases.
+    ///
```

Comment:
it could be clarified how these accounts with non-zero `locked` balances are called in staking pools (if there's any specific terminology). Are they the staking pools contract accounts themselves? 

or how they are called within consensus protocol, if that's the one.

Reply:
It's staking pool contracts themself